### DISCUSSION TITLE
新http模块获取的数据变成多个单字符

### DISCUSSION THREAD

**Discussion by nanhui1122** [labring/FastGPT#702] [Q&A]
使用http模块时，传入的数据是正确的json格式，但是收到后变成每个数字：单字符
![image](https://github.com/labring/FastGPT/assets/39914497/8f12d4e0-5507-42a9-a948-96f5e53db9f2)

**Comment by aohuajiaju** [comment]
fastgpt添加知识库显示系统不兼容，是什么情况

**Comment by Richerlv** [comment]
大佬，我也遇到了，这个问题有解决吗

**Reply by nanhui1122** [reply]
别用嵌套试试

**Comment by c121914yu** [comment]
没返回json?

**Reply by Richerlv** [reply]
是json，但是传入的每个字符都被拆开了
